addons/ai_apps/static/src/chat_window_patch.js

```diff
@@ -0,0 +1,30 @@
+import { ChatWindow } from "@mail/core/common/chat_window";
+
+import { useChildSubEnv } from "@odoo/owl";
+import { patch } from "@web/core/utils/patch";
+
+
+patch(ChatWindow.prototype, {
+    setup(){
+        super.setup();
+        let composerText;
+        if (this.thread?.messages.length === 0) {
+            composerText = this.props.chatWindow.composerText;
+        } else {
+            composerText = "";
+        }
+        useChildSubEnv({
+            insertToEditor: this.props.chatWindow.insert,
+            composerPreText: composerText,
+            chatCaller: this.props.chatWindow.chatCaller,
+        })
+    },
+    get style() {
+        const res = super.style;
+        if (this.thread?.channel_type !== "ai_composer") {
+            return res;
+        } else {
+            return res + ' z-index: 1056;';
```

Comment:
This looks useless, no? That's the same value s chatWindow default z-index defined in SCSS.

Reply:
I believe the default value is 1020 which would drop it below the full mail composer dialog.

Reply:
You could sync with the appropriate variable so the semantics is made obvious.

I would suggest to put a custom classname modifier on the chat window, by turning the `t-att-class` value into an overridable function `get attClass`.

There you could put `'o-isAiComposer': this.thread?.channel_type === "ai_composer"`, and define a CSS rule to put `z-index: $zindex-modal + 1`.

Reply:
Roughly something like this: https://github.com/odoo/odoo/compare/master...odoo-dev:odoo:master-ai-aku?expand=1